liblouis/compileTranslationTable.c

```diff
@@ -698,43 +698,19 @@ putCharAndDots (FileInfo * nested, widechar c, widechar d)
   return 1;
 }
 
+/**
+ * Print out dot numbers
+ */
 static char *
 unknownDots (widechar dots)
 {
-/*Print out dot numbers */
   static char buffer[20];
```

Comment:
Maybe a little comment saying what the longest possible output is, namely `\123456789ABCDEF/`?

Reply:
Couldn't is be `\123456789ABCDEF0/`?

Reply:
No, at least not with the old code.

Reply:
OK, you're right. With the new code it also checks for `B16` so the longest string could contain `0`. I think that should be ok

Reply:
Actually it's not OK, see my other comment.

Reply:
OK, should be the same as the old code now